ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -185,41 +185,43 @@
     >
       <F.Legend>{{t 'resources.target.workers.title'}}</F.Legend>
       <F.HelperText>{{t 'resources.target.workers.description'}}</F.HelperText>
-      {{#if this.showDeprecationMessage}}
+      {{#unless this.updateDeprecatedWorkerFilter}}
+        {{#if this.showDeprecationMessage}}
+          <F.Control>
+            <Hds::Alert @color='highlight' @type='inline' as |A|>
+              <A.Title>{{t
+                  'resources.target.workers.deprecation_message.title'
+                }}</A.Title>
+              <A.Description>{{t
+                  'resources.target.workers.deprecation_message.description'
+                }}</A.Description>
+            </Hds::Alert>
+          </F.Control>
+        {{/if}}
         <F.Control>
-          <Hds::Alert @color='highlight' @type='inline' as |A|>
-            <A.Title>{{t
-                'resources.target.workers.deprecation_message.title'
-              }}</A.Title>
-            <A.Description>{{t
-                'resources.target.workers.deprecation_message.description'
-              }}</A.Description>
-          </Hds::Alert>
+          <Hds::Form::TextInput::Field
+            @isRequired={{false}}
+            @isOptional={{true}}
+            @value={{@model.worker_filter}}
+            @isInvalid={{@model.errors.worker_filter}}
+            @type='text'
+            name='worker_filter'
+            disabled={{true}}
+            {{on 'input' (set-from-event @model 'worker_filter')}}
+            as |F|
+          >
+            <F.Label>{{t 'form.worker_filter.label'}}</F.Label>
+            <F.HelperText>{{t 'form.worker_filter.help'}}</F.HelperText>
+            {{#if @model.errors.worker_filter}}
+              <F.Error as |E|>
+                {{#each @model.errors.worker_filter as |error|}}
+                  <E.Message>{{error.message}}</E.Message>
+                {{/each}}
+              </F.Error>
+            {{/if}}
+          </Hds::Form::TextInput::Field>
```

Comment:
It looks like this `worker_filter` field needs to be nested with the {{#if this.showDeprecationMessage}}` as well. It is showing up when creating a new target. Another small changes is we can remove the error portion. Since it is always disabled we don't need to worry about errors on that field.